src/graphql/queries/SessionEvaluationPublicQuery.graphql

```diff
@@ -2,15 +2,6 @@ query EvaluateSessionPublic($sessionId: ID!) {
   sessionPublic(id: $sessionId) {
     id
     status
-    feedbacks {
```

Comment:
The public version of the evaluation should never include feedbacks or confusion.